docs/rules/prefer-to-be-defined.md

```diff
@@ -0,0 +1,28 @@
+# Suggest using  `toBeDefined()` (prefer-to-be-defined)
+
+In order to have a better failure message, `toBeDefined()` should be used upon asserting expections on defined value.
+
+## Rule details
+
+This rule triggers a warning if `toBe()` is used to assert a undefined value.
+
+```js
+expect(true).not.toBe(undefined);
+```
+
+This rule is enabled by default.
+
+### Default configuration
+
+The following patterns are considered warning:
+
+```js
+expect(true).not.toBe(undefined);
```

Comment:
I don't think this should warn, that should be caught by the other rule.

As eslint fixes are multipass, you'd get the same result in the end when autofixing, and then you don't have 2 rules warning about the same thing at once 🙂